x-pack/plugins/apm/server/routes/alerts/rule_types/transaction_duration/register_transaction_duration_rule_type.ts

```diff
@@ -140,26 +145,33 @@ export function registerTransactionDurationRuleType({
                 ...getDocumentTypeFilterForTransactions(
                   searchAggregatedTransactions
                 ),
-                { term: { [SERVICE_NAME]: ruleParams.serviceName } },
-                {
-                  term: {
-                    [TRANSACTION_TYPE]: ruleParams.transactionType,
-                  },
-                },
+                ...termQuery(SERVICE_NAME, ruleParams.serviceName),
```

Comment:
Can you test if the "all" option works as expected and is displayed as expected in the UI?